server/dataset_context.cc

```diff
@@ -129,6 +129,14 @@ doGetColumn(const Utf8String & tableName,
     throw HttpReturnException(400, "Cannot read column \"" + columnName.toUtf8String() + "\" with no dataset.");
 }
 
+GetAllColumnsOutput
+SqlExpressionMldbScope::
+doGetAllColumns(const Utf8String & tableName,
+                function<ColumnName (const ColumnName &)> keep)
+{
+    throw HttpReturnException(400, "Cannot get all columns with no dataset.");
```

Comment:
Also, the doGetColumn below should say "cannot read column "<columnName>" with no from clause"